src/thank_you_wonder_mail.c

```diff
@@ -88,9 +105,9 @@ extern const u8 DontSendItem_Text[];
 
 const MenuItem gUnknown_80DED78[3] =
 {
-    {"Confirm", 0xB},
-    {"Info", 0xC},
-    {NULL, 0x0},
+    {"Confirm", CONFIRM_ACTION},
+    {"Info", INFO_ACTION},
+    {NULL, CANCEL_ACTION},
```

Comment:
I wonder if it doesn't make more sense to make these last terminator entries use empty `{}` initializers.

Reply:
While doing that, the explicit array size could be removed.

Reply:
Was this comment addressed?

Reply:
Just did.